.github/workflows/_build_artifacts.yml

```diff
@@ -216,13 +216,24 @@ jobs:
             PROFILE=""
           fi
 
-          cross build $PROFILE -p ${{ matrix.name.package }} --target ${{ matrix.arch.target }}
+          # Hack to allow PR #4516 to pass CI
+          # Remove all this once it merges to main.
+          if [ "${{ matrix.name.package }}" = "firezone-headless-client" ] && [ -f "linux-client/Cargo.toml" ]
+          then
+            # We're building a Linux Client from before the PR, use the old name
+            PACKAGE=firezone-linux-client
+          else
+            # We're building another package, or it's after the PR, use the name as specified
+            PACKAGE=${{ matrix.name.package }}
+          fi
```

Comment:
I ran into this problem making manual build scripts for another project once.

I ended up with the "build script" being a wrapper that checks out the specifed commit and then builds the project using a commit-specific script from the specified version.

So I think if this was a `.sh` and we ran that in the context of the checked-out commit it'd work. But since the old commit is also in yml. it would not have worked this time. We could prepare that kind of thing for next time.